frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionDetails.tsx

```diff
@@ -37,22 +55,41 @@ function ModelActionDetails({ actions }: Props) {
     );
   }
 
+  const modelHasImplicitActions = hasImplicitActions(actions);
+
   return (
-    <ul>
-      {actions.map(action => (
-        <li key={action.id}>
-          <ActionListItem to={`/action/${action.action_id}`}>
-            <Icon name="insight" />
-            <ActionTitle>{action.name ?? humanize(action.slug)}</ActionTitle>
-          </ActionListItem>
-        </li>
-      ))}
-    </ul>
+    <>
+      {!modelHasImplicitActions && modelId && (
+        <Button onClick={createImplicitActions} icon="add">
+          {t`Enable implicit actions`}
+        </Button>
+      )}
+      <ul>
+        {actions.map(action => (
+          <li key={action.id}>
+            <ActionListItem
+              to={`/action/${action.id}`}
+              disabled={isImplicitAction(action)}
+            >
+              <Icon name="insight" />
+              <ActionTitle>
+                {action.name ?? humanize(action.slug ?? "")}
+              </ActionTitle>
+            </ActionListItem>
+          </li>
+        ))}
+      </ul>
+    </>
   );
 }
 
-export default Actions.loadList({
-  query: (state: State, props: { modelId?: number | null }) => ({
-    "model-id": props?.modelId,
-  }),
-})(ModelActionDetails);
+export default _.compose(
+  Actions.loadList(
+    {
+      query: (state: State, props: { modelId?: number | null }) => ({
+        "model-id": props?.modelId,
+      }),
+    },
+    connect(null, mapDispatchToProps),
+  )(ModelActionDetails),
+);
```

Comment:
The `compose` here doesn't seem to do anything, mind breaking the inner part down?